BusRoutes/ViewModel/RouteDeparturesViewModel.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using BusRoutes.Model;
+using BusRoutes.Service;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace BusRoutes.ViewModel
+{
+	public class RouteDeparturesViewModel:ViewModelBase
+	{
+		string title;
+		public string Title {
+			get {
+				return title;
+			}
+			set {
+				title = value;
+				Notify ();
+			}
+		}
+
+		Route currentRoute;
+		public Route CurrentRoute {
+			get {
+				return currentRoute;
+			}
+			set {
+				currentRoute = value;
+			}
+		}
+
+		ObservableCollection<Departure> departures;
+		public ObservableCollection<Departure> Departures {
+			get {
+				return departures;
+			}
+			set {
+				departures = value;
+				Notify ();
+			}
+		}
+
+		bool isLoading;
+		public bool IsLoading {
+			get {
+				return isLoading;
+			}
+			set {
+				isLoading = value;
+				Notify ();
+			}
+		}
+
+		public RouteDeparturesViewModel (Route route, DepartureSummary departureSummary)
+		{			
+			CurrentRoute = route;
+			Title = string.Format ("Timetable - {0} - {1}", route.ShortName, departureSummary.DayType);
```

Comment:
Avoid literal strings floating around, create constants for those, or better yet, have a string resources file where you can access them in a strong typed manner